src/lib.rs

```diff
@@ -1120,13 +1178,15 @@ mod tests {
 
 // The "standard library"
 use crate as sylt;
-sylt_macro::extern_function!(
-    dbg
-    [x] -> Type::Void => {
-        println!("DBG: {:?}, {:?}", x, Type::from(x));
-        Ok(Value::Nil)
-    },
-);
+
+pub fn dbg(values: &[Value], _typecheck: bool) -> Result<Value, ErrorKind> {
+    println!("{}: {:?}, {:?}",
+        "DBG".purple(),
+        values.iter().map(Type::from).collect::<Vec<_>>(),
+        values
+    );
+    Ok(Value::Nil)
+}
```

Comment:
This was a godsent when debugging. It helped a lot, since it also prints what the typesystem is doing.